metricflow/model/validations/validator_helpers.py

```diff
@@ -357,12 +357,24 @@ class DimensionInvariants:
 class ModelValidationRule(ABC):
     """Encapsulates logic for checking the values of objects in a model."""
 
-    @staticmethod
+    @classmethod
     @abstractmethod
-    def validate_model(model: UserConfiguredModel) -> List[ValidationIssueType]:
+    def validate_model(cls, model: UserConfiguredModel) -> List[ValidationIssueType]:
         """Check the given model and return a list of validation issues"""
         pass
 
+    @classmethod
+    def validate_model_serialized_for_multiprocessing(cls, serialized_model: str) -> str:
+        """Validate a model serialized via Pydantic's .json() method, and return a list of JSON serialized issues
+
+        This method exists because our warehouse validations are forked into parallel processes via
```

Comment:
Comment information fix: these are considered `semantic validations` not `warehouse validations` as these validation rules aren't run against the data warehouse.